mlflow/server/js/src/Actions.js

```diff
@@ -101,10 +101,10 @@ const wrapDeferred = (deferred, data) => {
       data,
       success: response => resolve(response),
       error: xhr => {
-        reject({ xhr })
+        console.error("XHR failed", xhr);
+        reject(new Error("XHR failed"));
```

Comment:
Just added an Error here as suggested by ESLint because I think this might allow downstream tools (such as the React dev environment) to display a stack trace correctly